Plugins/Bitbucket/BitbucketPullRequestForm.Designer.cs

```diff
@@ -777,5 +803,8 @@ private void InitializeComponent()
         private System.Windows.Forms.Label lblPRState;
         private System.Windows.Forms.Label label12;
         private System.Windows.Forms.Label label16;
+        private System.Windows.Forms.LinkLabel createPullLinkLabel;
+        private System.Windows.Forms.LinkLabel viewPullLinkLabel;
```

Comment:
we seem to be fans of completely opposite naming schemes - I start with control type prefix (e.g. `llblViewPull`), you fancy suffixes instead :)
not an action item, just an observation

Reply:
I have no strong preference myself, I try to follow the existing standard.
The normal in GE is the suffix, except for this module...
I will change these.

I should maybe change a few of the other like "label8" too, but in a separate PR.